app/propsGenerators/indexPagePropsGenerator.ts

```diff
@@ -65,13 +68,24 @@ function generateAllDhbsVaccineDoseData(
     if (southIslandDhbNames.includes(dhb.dhbName)) {
       regions.push("southIsland");
     }
+
+    const yesterdayDoseData = yesterdayVaccineData.find(
+      (yesterdayDhb) => yesterdayDhb.dhbName === dhb.dhbName
+    );
+
     return {
       ...dhb,
       regionIds: regions,
       hasMetFirstDoseTarget:
         dhb.firstDosesPercentage >= CONSTANTS.firstDoseTargetPercentage,
       hasMetSecondDoseTarget:
         dhb.secondDosesPercentage >= CONSTANTS.secondDoseTargetPercentage,
+      firstDosesChange: yesterdayDoseData
+        ? dhb.firstDosesTo90Percent - yesterdayDoseData?.firstDosesTo90Percent
+        : 0,
+      secondDosesChange: yesterdayDoseData
+        ? dhb.secondDosesTo90Percent - yesterdayDoseData?.secondDosesTo90Percent
```

Comment:
Maybe this should come from secondDoses? Also just thinking it should be a positive value 🤔 current designs are neg symbol, but it could be positive in the future too 🤔 

```suggestion
        ? dhb.secondDoses - yesterdayDoseData?.secondDoses
```

Reply:
Yep good point. I did go this route but went this way instead. Changed to be be positive and leave to symbol up to the UI to render